### DISCUSSION TITLE
As iter increases, both the absolute value and the fluctuation of rmse_f_trn become larger

### DISCUSSION THREAD

**Discussion by wqchen007** [deepmodeling/dpgen#659] [Q&A]
Dear users,
    I am using dpgen to explore training data, and I find that as iter increases, both the absolute value and the fluctuation of rmse_f_trn become larger. For iter.00 it is <0.1 and fluctuates around 0.1, but for iter.29 it fluctuates between 0.2~0.5. When doing the dp-test for the same database, RMSE for iter.29-model is also bad than earlier model.
    I check the data added to training base every iter and they are normally convergent. I tried to use more hidden layers and nodes, there is no significant improvement. I also tried to train with more steps, and dp-test results did not improve significantly either. I wonder whether this phenomenon is normal and what I can do to improve it.
    Besides, when I used the dp_model in large-scale MD, a large number of H atoms agglomerated (more than 2) and some bond length of H-H is smaller than theoretical value. I alse add some multi-hydrogen system data to training base but have no significant improvement. Should I continue to increase the number of relevant configurations for training? Is it right to add the problem configurations to the training set in order to make network learn that this configuration is not reasonable? I would be very grateful if there are other suggestions. 
    Thanks a lot!

**Comment by AnguseZhang** [comment]
What’s the difference between iter0 and iter29? Are the pressures and temperatures the same?

**Reply by wqchen007** [reply]
Thanks for your reply! In model_devi part, I first use the NVT then NPT to explore the structures. The data added to trainingbase from iter0 to iter29 consists of dpgen-fp structures(include NPT/NVT) and some designed structures. For the iter that the absolute value and the fluctuation of rmse_f_trn show a significant increase, I try to remove the corresponding data and retrain the model, but have no significant improvement. For the large-scale MD, all parameters (including the P or T) except the model are the same.

**Reply by AnguseZhang** [reply]
To be honest, your description for the protocol you used in DP-GEN is still not clear to me. One example is: "To better illustrate the idea, we take the copper system as an example, for which the user targets at a uniform accuracy over the thermodynamic range of temperature from 0 K to 2Tm and pressure from 1 Bar to 50,000 Bar, where Tm = 1357.77 K de- notes the melting point at the ambient condition. The exploration strategy can be designed as follows. We divide the temperature range [50 K,2Tm) equally into four intervals, i.e. [50 K,0.5Tm), [0.5Tm,Tm), [Tm,1.5Tm), [1.5Tm,2Tm), and explore them succes- sively. In each temperature interval we run 8 iterations with increasing number of MD simulations and increasing length of trajectories. For example in the first iteration, 600 MD simulations of 2 ps are performed, while in the last iteration the number of MD simulation increases to 2400 and the length trajectory increases to 6 ps. In each iteration, 5 temperatures conditions that evenly divide the temperature interval and 8 pressure conditions, i.e. 1, 10 100, 1000, 5000, 10,000, 20,000 and 50,000 Bar, are explored simultaneously by Isothermal–isobaric (NPT) DP-based MD simulations. The initial configurations of these MD simula- tions are prepared by randomly perturbing fully relaxed standard crystal structures, including face-centered cubic (fcc), hexagonal- closed-packed (hcp), and body-centered cubic (bcc) structures."
<img width="804" alt="Screen Shot 2022-03-13 at 3 49 37 AM" src="https://user-images.githubusercontent.com/38939321/158032791-f5c8cc5e-005d-40bd-8f67-2c7a2118cdfd.png">

**Comment by sooyaaa233** [comment]
I have also encountered this problem. Have you solved it?
